src/infrastructure/what-is-Nym.md

```diff
@@ -1 +1,30 @@
 # What is Nym?
+
```

Comment:
can you just remove this page from the commit? don't see what's changed but feel like there was a tweak or two that this might overwrite